io.tether.QVAC.Workbench.yml

```diff
@@ -0,0 +1,41 @@
+id: io.tether.QVAC.Workbench
+runtime: org.gnome.Platform
+runtime-version: '49'
+sdk: org.gnome.Sdk
+command: qvac_workbench
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=all
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+  - --talk-name=org.freedesktop.ScreenSaver
+modules:
+  - name: assets
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/share/metainfo/
+      - mkdir -p /app/share/icons/hicolor/512x512/apps/
+      - mkdir -p /app/share/applications/
+      - cp -r assets/. /app/bin/
+      - cp io.tether.QVAC.Workbench.metainfo.xml /app/share/metainfo/
+      - cp io.tether.QVAC.Workbench.png /app/share/icons/hicolor/512x512/apps/
+      - cp io.tether.QVAC.Workbench.desktop /app/share/applications/
+    sources:
+      - type: dir
+        path: .
+  - name: qvac_workbench-x64
+    only-arches:
+      - x86_64
+    buildsystem: simple
+    build-commands:
+      - cp qvac_workbench $FLATPAK_DEST/bin
+    sources:
+      - type: archive
+        url: https://static.tether.dev/qvac/workbench/QVAC.Workbench.linux.x64.bin.tar.gz
```

Comment:
This should be built from source.

https://docs.flathub.org/docs/for-app-authors/requirements#building-from-source

Reply:
What does the license have to do with whether we release the source code now or later? There isn't even an obligation to do so, even if binaries end up being packaged with the license itself. What am I missing?